paasta_tools/kubernetes_tools.py

```diff
@@ -1604,14 +1610,13 @@ def get_boto_new_secret_signature_name(self) -> str:
     def get_boto_secret_signature_name(self) -> str:
         """
         Keep the following signature naming convention so that bounces do not happen because boto_keys configmap signatures already exist, see PAASTA-17910
-        Most services reside in paasta namespace so we can pull it dynamically
 
-        Note: Since hashing is done only portion of secret, it may explode if service or namespace too long, so we should switch to get_boto_new_secret_signature_name as new namespaces are rolled out
+        Note: Since hashing is done only portion of secret, it may explode if service or instance names are too long, so we should switch to get_boto_new_secret_signature_name as new namespaces are rolled out
```

Comment:
imo, we probably want to move away from including the namespace in the secret name as it's redundant (and there's also currently no toggle for changing the current behavior by-namespace)

additionally, we could/should (in a future PR) update the return in this function to limit the signature name's size with `limit_size_with_hash` to avoid the scenario mentioned in the docstring :)

Reply:
```suggestion
        Note: Since hashing is done only portion of secret, it may explode if service or instance names are too long
```